awx/main/models/host_metrics.py

```diff
@@ -0,0 +1,6 @@
+from django.db import models
+
+class HostMetrics(models.Model):
+    hostname = models.CharField(primary_key=True,max_length=30)
+    first_automation = models.DateTimeField()
```

Comment:
It might make sense to just have a default here that gets set on insert e.g., `default=django.utils.timezone.now`

Reply:
I have updated it